apps/api/app/services/todos/todo_bulk_service.py

```
@@ -187,15 +221,38 @@ async def bulk_delete_todos(todo_ids: List[str], user_id: str) -> None:
+                "user_id": user_id,
+                "vfs_path": {"$exists": True, "$ne": None},
+            },
+            {"_id": 1, "vfs_path": 1},
+        )
+        tracked_docs = await tracked_cursor.to_list(length=None)
+        for doc in tracked_docs:
+            tid = str(doc["_id"])
+            try:
+                await delete_canvas_embedding(tid)
+            except Exception:
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)